scripts/cloud-hooks/functions.sh

```diff
@@ -8,12 +8,13 @@ drush_alias=${site}'.'${target_env}
 deploy_updates() {
 
   case $target_env in
-    01dev|01test|01live)
+    01dev|01test)
       acsf_deploy
       ;;
-    01devup|01testup|01update)
+    # Do not run deploy updates on 01live in case a branch is deployed in prod. 
+    01devup|01testup|01update|01live)
       ;;
-    ode[[:digit:]]*)
```

Comment:
Did you intend to remove the ode check here? Isn't this going to cause all deployments to ACE to install the site from scratch?